tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import argparse
+import requests
+from bs4 import BeautifulSoup
+
+
+def gets_parsed_html_from_page(url):
```

Comment:
I think that in Python if you've got methods that don't get called from outside the file, you can mark them up as "private" with a cheeky underscore prefix?  Might be worth doing here, so we know what the main methods are to call from outside this script?